packages/application-components/src/index.ts

```diff
@@ -44,7 +44,6 @@ export type { TPageContentFull } from './components/page-content-containers/page
 
 // Custom views
 export { default as CustomPanel } from './components/custom-views/custom-panel';
-export { default as CustomViewLoader } from './components/custom-views/custom-view-loader';
```

Comment:
I decided to move it to `application-shell` package, as it does have some logic depending on the application context and it's easier to test it from within the application-shell package.

Users won't use this component directly anyway, so it's only for us internally.